src/assets/tech_ships.yaml

```diff
@@ -0,0 +1,179 @@
+tech:
```

Comment:
What do you think if we separate this file in two?
One for techs and another for ships.

I love this so much.

Reply:
I've gone back and forth on that.  Two files would probably work well.

One thing I was planning on doing is having sub-sections for "additional" techs/units from the expansions, depending on which production sheet you use.  But that would work with one file or two files.

I will split the techs/ships into to files and update the PR.